addons/website/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,72 @@
+---Fields in module 'website'---
+website      / ir.ui.view               / page (boolean)                : DEL
+#DONE: Copy ir.ui.view pages to website.page
+
+website      / website                  / auto_redirect_lang (boolean)  : NEW
+website      / website                  / compress_html (boolean)       : DEL
+website      / website                  / google_management_client_id (char): NEW
+website      / website                  / google_management_client_secret (char): NEW
+website      / website                  / homepage_id (many2one)        : NEW relation: website.page
+website      / website.menu             / page_id (many2one)            : NEW relation: website.page
+website      / website.page             / _inherits (False)             : NEW
```

Comment:
Put all the website.page lines together with the first line because all these changes are covered in the same method